util/src/main/scala/scala/scalanative/io/ByteBufferPool.scala

```diff
@@ -7,7 +7,7 @@ final class ByteBufferPool {
   private var buffers: List[ByteBuffer] = Nil
 
   private def alloc(): ByteBuffer = {
-    ByteBuffer.allocateDirect(64 * 1024 * 1024)
+    ByteBuffer.allocateDirect(8 * 1024 * 1024)
```

Comment:
The largest `.nir` file I could find in the repo is 2M so yes this is somewhat of a stopgap measure, but fixing the NIR code generation is a much larger effort to refactor.

```
> find . -name "*.nir" | xargs stat -f "%z %N" | sort -n | tail -5
    546211 ./scalalib/target/scala-2.11/classes/scala/collection/immutable/StreamViewLike$$anon$13.nir
    580412 ./scalalib/target/scala-2.11/classes/scala/collection/mutable/IndexedSeqLike$$anon$1.nir
    612962 ./javalib/target/scala-2.11/classes/java/math/BigDecimal.nir
    699387 ./javalib/target/scala-2.11/classes/java/util/Arrays$.nir
    2301253 ./javalib/target/scala-2.11/classes/java/lang/Character$.nir
```